cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -144,7 +144,7 @@ impl CatalogClient {
         }
 
         let client = {
-            let timeout = std::time::Duration::from_secs(15);
+            let timeout = std::time::Duration::from_secs(60);
```

Comment:
question blocking: my networking knowledge isn't very strong, but do we need the connect timeout to be so high?

Reply:
Nope, not intentional.  Probably good to address that.  No need to wait 60s if you can't even connect.

Reply:
Fixed.